src/components/ReportActionItem/ReportPreview.js

```diff
@@ -123,6 +123,8 @@ function ReportPreview(props) {
     const transactionsWithReceipts = ReportUtils.getTransactionsWithReceipts(props.iouReportID);
     const numberOfScanningReceipts = _.filter(transactionsWithReceipts, (transaction) => TransactionUtils.isReceiptBeingScanned(transaction)).length;
     const hasReceipts = transactionsWithReceipts.length > 0;
+    const distanceRequestTransactions = ReportUtils.getDistanceRequestTransactions(props.iouReportID);
```

Comment:
If we still retain two separate calls, why not use `ReportUtils.hasOnlyDistanceRequestTransactions` ?

Reply:
I was a bit concerned about checking this value against the report transactions (which is done inside `ReportUtils.hasOnlyDistanceRequestTransactions`), while all the rest here are compared with `numberOfRequests`, which is taken from `reportPreviewAction.childMoneyRequestCount`.

But looks like we are safe to use the `ReportUtils.hasOnlyDistanceRequestTransactions` here indeed.